bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
Ah, it seems that in Python 2 arguments are merely `str` (not `unicode`), while in Python 3 they are Unicode `str`, so we have to convert them into unicode first.